archapp/interactive.py

```diff
@@ -80,7 +80,33 @@ def __init__(self, hostname=config.hostname,
         {host}
         {data}
         {mgmt}
+
+        Methods
+        -------
+        get
+            Return timeseries data from the archiver as an xarray or numpy array.
+
+        plot
+            Plot timeseries data from the archiver in a new figure.
+
+        prints
+            Print timeseries data from the archiver.
+
+        search
+            Queries the archiver with a PV search using glob format.
+
+        Examples
+        --------
+        To get data, use:
+        >>> data = archive.get("pv_name", xarray=True)
+
+        To get the data in a data frame format:
+        >>> data.to_dataframe()
```

Comment:
I thought this was the default now:
```suggestion
        >>> data = archive.get("pv_name")
```

Reply:
Oh never mind, disregard me, the default gives you a numpy array